editor/RubyScanner.cpp

```diff
@@ -140,7 +140,7 @@ bool Scanner::didBlockStart()
 
 bool Scanner::didBlockEnd()
 {
-    static const QRegExp regex(QLatin1String("24_"));
+    static const QRegExp regex(QLatin1String("(24|29)_"));
```

Comment:
This is actually intentional. It is meant to cover the following case:

    if (a == 4 ||
      b == 8) # indent here...

But it does break the following case:

    if (a == 4)
    # no indentation here...

Can you please look into this?